src/secret_sharing/xor.rs

```diff
@@ -1,10 +1,14 @@
-#[derive(Clone, Copy, Debug)]
+use std::io;
+
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
 pub struct XorReplicated {
     left: u64,
     right: u64,
 }
 
 impl XorReplicated {
+    pub const SIZE_IN_BYTES: usize = 2 * std::mem::size_of::<u64>();
```

Comment:
I think that you want to look at Taiki's latest in #409 here.  And isn't this supposed to attach to an implementation of `Message`?

Reply:
I was a little cautious with that trait because it kinda carries two meanings - something that is serializable and something that can be sent between helpers. I am thinking that I should probably create a supertrait to `Message` and make xor share implement it?

Reply:
Either way works for me.  I had the same thought, but I couldn't bring myself to care.